neuralop/training/callbacks.py

```diff
@@ -401,42 +401,23 @@ def on_train_start(self, *args, **kwargs):
         if self.resume_from_dir:
             saved_modules = [x.stem for x in self.resume_from_dir.glob("*.pt")]
 
-            assert (
-                "best_model_state_dict" in saved_modules
-                or "model_state_dict" in saved_modules
-            ), "Error: CheckpointCallback expects a model state dict named model.pt or best_model.pt."
-
             # no need to handle exceptions if assertion that either model file exists passes
             if "best_model_state_dict" in saved_modules:
```

Comment:
I would just build the path and then use if `path.exists()` (using pathlib)

Reply:
done